std/Codata/CoMaybe.duo

```diff
@@ -0,0 +1,79 @@
+module Codata.CoMaybe;
+
+import Codata.Function;
+import Data.Cofunction;
+import Data.NegationP;
+import Data.Bool;
+import Prim.I64;
+
+-- | The Maybe type encapsulates optional values.
```

Comment:
We probably shouldn't duplicate Data.Maybe here.

Reply:
ah, hab ich das aus Versehen commitet?
Kann raus.